bigchaindb/processes.py

```diff
@@ -70,19 +70,18 @@ def start(self):
         p_map_bigchain = mp.Process(name='bigchain_mapper', target=self.map_bigchain)
         p_block_delete_revert = mp.Process(name='block_delete_revert', target=delete_reverter.start)
         p_voter = Voter(self.q_new_block)
-        p_election = Election(self.q_block_new_vote)
```

Comment:
since we are no longer using `q_block_new_vote` we can remove it from `Processes.__init__` and the `update` part in `map_bigchain`.